helmfile.d/values/hnc/controller.yaml.gotmpl

```
@@ -1,15 +1,35 @@
 includedNamespacesRegex: {{ .Values.hnc.includedNamespacesRegex }}
 
 excludedNamespaces:
-  {{- with .Values.hnc.excludedNamespaces }}
-  {{- toYaml . | nindent 2 }}
-  {{- end }}
   {{- with .Values.hnc.excludedExtraNamespaces }}
   {{- toYaml . | nindent 2 }}
   {{- end }}
   {{- with (keys .Values.user.constraints) }}
   {{- toYaml . | nindent 2 }}
   {{- end }}
+  - alertmanager
+  - cert-manager
+  - falco
+  - fluentd
+  - fluentd-system
+  - hnc-system
+  - ingress-nginx
+  - kube-node-lease
+  - kube-public
+  - kube-system
+  - kured
+  - monitoring
+  - rook-ceph
+  - velero
+  - gatekeeper-system
+  - metallb-system
+  {{- if eq .Values.global.ck8sK8sInstaller "capi" }}
+  - calico-apiserver
+  - calico-system
+  - openstack-system
+  - gpu-operator
+  - tigera-operator
+  {{- end }}
```

Was there a reason why we didn't just reserve these always?

```suggestion
  - calico-apiserver
  - calico-system
  - openstack-system
  - gpu-operator
  - tigera-operator
```

---

Right, I'll fix that :+1: